### ISSUE TITLE
Segmentation fault when cursor is too low

### ISSUE THREAD

**Issue by noughtmare** [jtdaugherty/vty#140]
The following code segfaults for me:

```haskell
import Graphics.Vty

main :: IO ()
main = do
  vty <- standardIOConfig >>= mkVty
  update vty ((picForImage mempty) {picCursor = Cursor 0 1000})
  nextEvent vty
  shutdown vty
```

I think the y coordinate of the cursor is not checked before it is passed to the underlying c code.

**Comment by jtdaugherty** [comment]
Thanks for the report!

**Comment by jtdaugherty** [comment]
This is now fixed in `5.19.2`. Thanks!
